sonic_platform_base/sonic_sfp/sfputilbase.py

```diff
@@ -1253,8 +1253,12 @@ def get_transceiver_change_event(self, timeout=0):
          event. When timeout is non-zero, the function can also return when the timer expires.
          When timer expires, the return status is True and events is empty.
         :returns: (status, events)
-        :status: Boolean, True if call successful, False if not;
-        :events: dictionary for pysical port index and the SFP status,
-        status='1' represent plug in, '0' represent plug out like {'0': '1', '31':'0'}
+        :status: Boolean, True if call successful and no system level event/error occurred, 
+         False if call not success or system level event/error occurred.
```

Comment:
What is the status value if call successful and system level event occurred?

Reply:
It would be False: "False if call not success **or system level event/error occurred**"